project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
\ No newline at end of file
+sbt.version=1.1.0
```

Comment:
@kgudel Could we please not change the SBT version as part of this PR?

Reply:
Added accidentally, removing.